packages/zone.js/package.json

```
@@ -42,5 +42,314 @@
   "license": "MIT",
   "bugs": {
     "url": "https://github.com/angular/angular/issues"
+  },
+  "exports": {
+    "./package.json": {
+      "default": "./package.json"
+    },
+    ".": {
+      "typings": "./zone.d.ts",
+      "import": "./fesm2015/zone.js",
+      "require": "./bundles/zone.umd.js",
+      "default": "./fesm2015/zone.js"
+    },
+    "./dist/*": {
+      "default": "./bundles/*.umd.js"
+    },
+    "./testing": {
+      "import": "./fesm2015/zone-testing.js",
+      "require": "./bundles/zone-testing.umd.js",
+      "default": "./fesm2015/zone-testing.js"
+    },
+    "./node": {
+      "import": "./fesm2015/zone-node.js",
+      "require": "./bundles/zone-node.umd.js",
+      "default": "./fesm2015/zone-node.js"
+    },
+    "./mix": {
+      "import": "./fesm2015/zone-mix.js",
+      "require": "./bundles/zone-mix.umd.js",
+      "default": "./fesm2015/zone-mix.js"
+    },
+    "./plugins/async-test": {
+      "import": "./fesm2015/async-test.js",
+      "require": "./bundles/async-test.umd.js",
+      "default": "./fesm2015/async-test.js"
+    },
+    "./plugins/async-test.min": {
+      "import": "./fesm2015/async-test.min.js",
+      "require": "./bundles/async-test.umd.min.js",
+      "default": "./fesm2015/async-test.min.js"
+    },
```

```suggestion
    "./plugins/*.min": {
      "import": "./fesm2015/*.min.js",
      "require": "./bundles/*.umd.min.js",
      "default": "./fesm2015/*.min.js"
    },
    "./plugins/*": {
      "import": "./fesm2015/*.js",
      "require": "./bundles/*.umd.js",
      "default": "./fesm2015/*.js"
    },
```
The subpath patterns should allow condensing the individual plugin entries into just two.